socorro/unittest/signature/test_rules.py

```diff
@@ -278,26 +279,20 @@ def test_generate_signature_3(self):
     def test_generate_signature_4(self):
         j = JavaSignatureTool()
         java_stack_trace = (
-            '   SomeJavaException: %s  \n'
-            'at org.mozilla.lars.myInvention(larsFile.java)' % ('t' * 1000)
+            '   SomeJavaException: t  \nat org.mozilla.lars.myInvention(larsFile.java)'
```

Comment:
I messed up--I need to re-fix this because it should be testing that the description gets dropped if it's too long.